src/renderer/dx/CustomTextLayout.cpp

```diff
@@ -1379,3 +1529,229 @@ void CustomTextLayout::_OrderRuns()
 }
 
 #pragma endregion
+
+// Routine Description:

```

Comment:
Should this and its friendos above join together in one region?

Reply:
sure, why not